src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +103,19 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        return someDate + (someDate.isBefore(LocalDate.now()) ? " is before " + LocalDate.now()
+                : someDate.isAfter(LocalDate.now()) ? " is after " + LocalDate.now()
```

Comment:
we user LocalDate.now() three times here, let's make it a local variable inside this method